core-web/libs/edit-content/tsconfig.json

```diff
@@ -3,9 +3,7 @@
         "target": "es2022",
         "useDefineForClassFields": false,
         "forceConsistentCasingInFileNames": true,
-        "strict": true,
```

Comment:
@fmontes do we want to deactivate strict mode? Shall we activate also `strictNullChecks`?

Reply:
I talked about this with freddy and we won't activate it because it fails while checking imports.